### ISSUE TITLE
Unable to add a book

### ISSUE THREAD

**Issue by knjk04** [knjk04/book-project#28]
**Describe the bug**

A new book cannot be added as the 'add book' button is greyed out.

**To Reproduce**
Steps to reproduce the behaviour:
1. Fill in the fields to add a new book (see screenshot)
2. See error

**Expected behaviour**

The add book button should not be greyed out and the book should be saved into the repository.

**Screenshots**

![cannot-add-new-book](https://user-images.githubusercontent.com/11173328/82117617-10f3e100-9769-11ea-9167-192f2eb3e2be.png)

**Additional context**

Branch: [ui](https://github.com/knjk04/book-project/tree/ui)

Please create your branch off [ui](https://github.com/knjk04/book-project/tree/ui) and submit a PR to merge into it when done.

**Comment by knjk04** [comment]
Also, the last name error message should say "Please enter a last name" when the field is empty.

**Comment by knjk04** [comment]
Commit [a0672a3](https://github.com/knjk04/book-project/commit/a0672a3df32043a4050fb4a1cdc4901a512da1fe) fixes the author validation, but a book can still not be added.

**Comment by knjk04** [comment]
Since commit [c421ee4](https://github.com/knjk04/book-project/commit/c421ee45ab25825fa7e0d8bcf01eb73ae9ff8a84), a new book can be added, but the author's first name is not bound. The other fields are correctly bound.

([c421ee4](https://github.com/knjk04/book-project/commit/c421ee45ab25825fa7e0d8bcf01eb73ae9ff8a84) changed the genre from a MultiComboBox to a ComboBox, and now the form field correctly binds.)

![null-author](https://user-images.githubusercontent.com/11173328/82475860-0e74ec80-9ac5-11ea-9bc5-c59275da193c.png)

**Comment by knjk04** [comment]
Blocked by issue #30
